examples/bri-3/prisma/migrations/20240124083845_add_public_key_to_bpi_subject/migration.sql

```diff
@@ -0,0 +1,27 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `publicKey` on the `BpiSubject` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "BpiSubject" DROP COLUMN "publicKey";
+
+-- CreateTable
+CREATE TABLE "PublicKey" (
+    "id" TEXT NOT NULL,
+    "type" TEXT NOT NULL,
+    "value" TEXT NOT NULL,
+    "bpiSubjectId" TEXT NOT NULL,
```

Comment:
all entries should be not NULL ... and should not `bpiSubjectId` be the foreign key for that table? @biscuitdey